airbyte-integrations/connectors/source-yandex-metrica/source_yandex_metrica/spec.yaml

```diff
@@ -0,0 +1,292 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Yandex Metrica Spec
+  type: object
+  required:
+    - auth_token
+    - counter_id
+    - start_date
+    - end_date    
+    - hits_fields
+    - visits_fields
```

Comment:
If a User only wants to sync visits it won't allow

Reply:
Removed field selection during setup. Fields are passed from predefined list now.